tests/AutoReview/ProjectCodeTest.php

```diff
@@ -339,6 +339,7 @@ private function getTestClasses()
             ->in(__DIR__.'/..')
             ->exclude(array(
                 'Fixtures',
+                'Test',
```

Comment:
this is used for data providing to determine that `tests/` classes are internal and final. it's not the case for `tests/Test`, as we expose it.
Actually, we also have `tests/Test/SthSthTest.php`, which shall be internal. I removed `Test` from excluding and enforce file to have `Test` suffix few lines above